src/components/DittoComponent.tsx

```diff
@@ -3,17 +3,21 @@ import { useDittoComponent } from "../hooks/useDittoComponent";
 import { DittoComponentLibraryProps } from "./Ditto";
 
 export const DittoComponent = (props: DittoComponentLibraryProps) => {
-  const { children, componentId } = props;
+  const { children, componentId, variables = {}, count } = props;
+
   const value = useDittoComponent({
     componentId,
     alwaysReturnString: typeof children !== "function",
+    variables,
+    count
   });
 
-  const text = useMemo(
-    () => (value !== null && typeof value === "object" ? value.text : value),
-    [value]
-  );
+  // const text = useMemo(
+  //   () => (value !== null && typeof value === "object" ? value.text : value),
+  //   [value, variables]
+  // );
```

Comment:
I think we can probably keep this as is, since `value` should update any time `variables` updates, which will in turn update the memoized `text` value

Reply:
👍